src/compression/snappy.rs

```diff
@@ -230,14 +256,14 @@ mod tests {
                 n => buf.extend_from_slice(&tmp_buf[..n]),
             }
         }
-        assert_eq!(ORIGINAL.as_bytes(), &buf[..]);
+        assert_eq!(ORIGINAL, str::from_utf8(&buf[..]).unwrap());
     }
 
     #[test]
     fn test_snappy_reader_read_to_end() {
         let mut buf = Vec::new();
         let mut r = SnappyReader::new(COMPRESSED).unwrap();
         r.read_to_end(&mut buf).unwrap();
-        assert_eq!(ORIGINAL.as_bytes(), &buf[..]);
+        assert_eq!(ORIGINAL, str::from_utf8(&buf[..]).unwrap());
```

Comment:
i guess, this change leads to nicer/more readable output if the assertions fails, correct?